keyboards/the_royal/liminal/config.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x6044
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    TheRoyalSweatshirt
+#define PRODUCT         Liminal
+#define DESCRIPTION     "A 50% keyboard with focused around spectacular keycap compatibility."
+
+/* key matrix size */
+#define MATRIX_ROWS 4
+#define MATRIX_COLS 14
+
+#define MATRIX_ROW_PINS { C6, B6, B7, C7 }
+#define MATRIX_COL_PINS { D6, C4, D3, D2, D1, D0, C2, B0, B1, B2, B3, B4, D5, C5 }
+#define UNUSED_PINS
+
+/* COL2ROW or ROW2COL */
+#define DIODE_DIRECTION COL2ROW
+
+/* Set 0 if debouncing isn't needed */
+#define DEBOUNCE 5
+
+#define PERMISSIVE_HOLD
+
+#define TAPPING_TERM 200
+
+#define TAPPING_TERM_PER_KEY
+
```

Comment:
```suggestion

```

Reply:
yeah Ill change it.  Ive always just like XXX or XXXX cause it helps keep the layers pretty even in terms of physical spacing